shell/browser/api/electron_api_push_notifications.cc

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) 2022 Asana, Inc.
+// Use of this source code is governed by the MIT license that can be
+// found in the LICENSE file.
+
+#include "shell/browser/api/electron_api_push_notifications.h"
+
+#include <string>
+
+#include "shell/common/gin_converters/value_converter.h"
+#include "shell/common/gin_helper/dictionary.h"
+#include "shell/common/node_includes.h"
+
+namespace electron {
+
+namespace api {
+
+PushNotifications* push_notifications = nullptr;
+
+gin::WrapperInfo PushNotifications::kWrapperInfo = {gin::kEmbedderNativeGin};
+
+PushNotifications::PushNotifications() {
+  DCHECK(!push_notifications);
+  push_notifications = this;
+}
+
+PushNotifications::~PushNotifications() {
+  DCHECK(push_notifications);
+  push_notifications = nullptr;
+}
+
+// static
+PushNotifications* PushNotifications::Get() {
+  return push_notifications;
+}
+
+// static
+gin::Handle<PushNotifications> PushNotifications::Create(v8::Isolate* isolate) {
+  return gin::CreateHandle(isolate, new PushNotifications());
```

Comment:
```suggestion
  return gin::CreateHandle(isolate, PushNotifications::Get());
```

Reply:
This change & the previous change together just add a little more safety, as the `PushNotifications::Create` function is public and could potentially be called multiple times. With these changes, calling `PushNotifications::Create` multiple times will just return multiple handles to the same object, which is fine.